G1ANT.Addon.Net/SmtpManager.cs

```diff
@@ -0,0 +1,56 @@
+﻿using MailKit.Net.Smtp;
+using MailKit.Security;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace G1ANT.Addon.Net
+{
+    public sealed class SmtpManager
+    {
+        private NetworkCredential credentials;
+        private SmtpClient client;
+        private string host;
+        private int port;
+        private SecureSocketOptions socketOptions = SecureSocketOptions.Auto;
+
+        public static SmtpManager Instance { get; } = new SmtpManager();
```

Comment:
What if you want to use two instances at the same time? Consider Manager/Wrapper pattern as used in some other addons.

Reply:
It has been developed in this way months ago. I'm not going to refactor whole addon :P